src/tests/unittests/instrument_drivers/virtualAwg/test_sequencer.py

```diff
@@ -127,8 +127,8 @@ def test_qupulse_template_to_array_new_style_vs_values_old_style(self):
 
             # the value to compare to are calculated using qupulse 0.4 Sequencer class
             self.assertTrue(1000001 == len(voltages_new['sawtooth']))
-            self.assertAlmostEqual(-amplitude/2, np.min(voltages_new['sawtooth']), 12)
-            self.assertAlmostEqual(amplitude/2, np.max(voltages_new['sawtooth']), 12)
+            self.assertAlmostEqual(-0.75, np.min(voltages_new['sawtooth']), 12)
```

Comment:
Remove this change, also on the next line

Reply:
I think @QFer did this change for me to be able to pass the pipeline

Reply:
@QFer and @peendebak , I checked out the module from dev and pushed it again.